.changes/unreleased/Added-20220213-200122.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Added
+body: allow for use of sprig functions in templates
```

Comment:
```suggestion
body: Allow for use of sprig functions in templates
```
I have been using capitals to start change fragments

Reply:
Fixed